polaris-react/src/components/Box/Box.tsx

```diff
@@ -74,31 +72,48 @@ interface Spacing {
 }
 
 export interface BoxProps {
-  as?: 'div' | 'span';
-  /** Background color of the Box */
+  /** Used to indicate the element is being modified */
+  ariaBusy?: boolean;
+  /** Used to identify the ID of the element that describes Box */
+  ariaDescribedBy?: string;
+  /** Used to identify the Id of element used as aria-labelledby */
+  ariaLabelledBy?: string;
+  /** Used to indicate the element will be updated */
+  ariaLive?: string;
+  /** Used to describe the semantic element type */
+  ariaRoleType?: string;
```

Comment:
How would you feel about not being explicit about these and moving them to a ...rest block? We could still be explicit in our typings to omit certain things like the style attr from being passed in.

I'd group aria into that block as well as the onClick / blur / keyup stuff as well.

Reply:
@kyledurand and I chatted offline about this, TypeScript complains about including any of the aria attributes or event handlers in rest so leaving as is for now.